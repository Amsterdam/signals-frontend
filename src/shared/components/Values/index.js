import * as React from 'react';

export default (value) => (
  <div style={styles.main}>
    <h3 style={{ color: '#389926' }}>Values</h3>
    <pre style={styles.text}>{JSON.stringify(value)}</pre>
  </div>
);

const styles = {
  main: {
    border: '1px solid #ccc',
    backgroundColor: '#000',
    borderRadius: '4px',
    width: '80%',
    marginLeft: '7%',
    marginTop: '20px',
    padding: '10px 20px',
    textAlign: 'left'
  },
  text: {
    color: '#DDE1DC',
    fontSize: '18px',
    lineHeight: '30px',
  }
};
