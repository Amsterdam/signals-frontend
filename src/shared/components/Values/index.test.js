import React from 'react';
import { mount, shallow } from 'enzyme';

import Values from './';

describe('<Values />', () => {
  it('should render an <div> tag', () => {
    const renderedComponent = mount(<Values />);
    expect(renderedComponent.find('div').length).toEqual(1);
  });

  it('should render the json ', () => {
    const value = { id: 1, name: 'name' };
    const renderedComponent = shallow(<Values value={value} />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
