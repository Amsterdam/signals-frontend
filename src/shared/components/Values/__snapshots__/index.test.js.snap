// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Values /> should render the json  1`] = `
<div
  style={
    Object {
      "backgroundColor": "#000",
      "border": "1px solid #ccc",
      "borderRadius": "4px",
      "marginLeft": "7%",
      "marginTop": "20px",
      "padding": "10px 20px",
      "textAlign": "left",
      "width": "80%",
    }
  }
>
  <h3
    style={
      Object {
        "color": "#389926",
      }
    }
  >
    Values
  </h3>
  <pre
    style={
      Object {
        "color": "#DDE1DC",
        "fontSize": "18px",
        "lineHeight": "30px",
      }
    }
  >
    {"value":{"id":1,"name":"name"}}
  </pre>
</div>
`;
